tools/test-backend

```diff
@@ -407,7 +413,6 @@ def main() -> None:
         cov.stop()
         cov.save()
         cov.combine()
-        cov.data_suffix = False  # Disable suffix so that filename is .coverage
```

Comment:
perhaps we should have the `data_suffix` change as a separate commit? I was confused for a minute about why this was necessary for the `no-cov-cleanup` option, before realizing that this is a conceptually separate change

Reply:
If it's necessary for this fix, we should keep them in same commit. Let me see their documentation about data_suffix and see if we can do better here.